source/features/release-download-count.tsx

```diff
@@ -59,9 +60,10 @@ async function addCounts(assetsList: HTMLElement): Promise<void> {
 	const releases = [[releaseName, assetsList]] as const;
 	for (const [name, release] of releases) {
 		const sortedDownloads = assets[api.escapeKey(name)].sort((a, b) => b.downloadCount - a.downloadCount);
```

Comment:
We're no longer using this sorted array. At a minimum we should drop the sorting
```suggestion
		const downloads = assets[api.escapeKey(name)];
```

Better yet if we avoid a loop below by making this a map:

```suggestion
		const downloads = new Map(assets[api.escapeKey(name)].map(asset => [asset.name, asset.downloadCount]))
```

And then later replace the `sortedDownloads` loop with:

```js
const downloadCount = downloads.get(assetName.textContent!)
if (!downloadCount) {
	continue;
}

Reply:
Nice, updated in latest.